src/Domain.php

```diff
@@ -20,6 +20,7 @@
 use Pdp\Exception\CouldNotResolveSubDomain;
 use Pdp\Exception\InvalidLabel;
 use Pdp\Exception\InvalidLabelKey;
+use Throwable;
```

Comment:
`Throwable` is no longer used in the class.

Reply:
you should use the following command lines
```
 ./vendor/bin/php-cs-fixer fix -vvv --allow-risky=yes
composer phpstan
```

Prior to committing your changes on your local machine these command will:

- fixes any potential coding style issue
- uses PHPStan to analyze and improve your code